doc/source/testing_pull_requests.rst

```diff
@@ -0,0 +1,178 @@
+Testing an python-for-android pull request
+==========================================
+
+In order to test a p4a pull requests we contemplate three possible scenarios:
+
+  - using python-for-android commands directly from the pull request files
+    that we want to test, without installing it (the recommended way for most
+    of the test cases)
+  - installing python-for-android using the github's branch of the pull request
+  - using buildozer and a custom app
+
+We will explain the first two methods using one of the distributed
+python-for-android test apps and we assume that you already have the
+python-for-android dependencies installed. For the `buildozer` method we assume that you
+already have a a properly working app to test and a working
+installation/configuration of `buildozer`. There is one step that it's shared
+with all the testing methods that we propose in here...we named it
+`Common steps`.
+
+
+Common steps
+^^^^^^^^^^^^
+
+- open a terminal, create a test directory and enter inside it, eg.::
+
+    .. codeblock:: bash
+
+        mkdir p4a_pull_requests
+        cd p4a_pull_requests
+
+- clone the pull request you wanna to test, let's imagine that you wanna to
+  test a pull request whose branch is named `feature-fix-numpy` and the github
+  nickname of the user who created the pull request is named `obiwankenoby`::
```

Comment:
Minor: Obi-Wan Kenobi is actually spelled with a `i` not a `y` :smile:

Reply:
Oh :scream:, How the hell I spelled wrong the **Obi-Wan**'s last name?
forgive me for this, this is major not minor :laughing:
I will definitively change that...

**¡¡¡Thanks!!!**

and

**May the Force be with you** :wink: